charts/polkadot/templates/alertrules.yaml

```diff
@@ -9,76 +9,4 @@ spec:
   groups:
   - name: polkadot.rules
     rules:
```

Comment:
This files should contain the rules defined by ourself. The fact is that all the rules we had are already defined in the new imported alertrules-substrate.yaml, so now this set of rule is empty ( but extendible )

Reply:
MAkes total sense :+1: